### PR TITLE
Task1 part2 implementation design

### PR DESCRIPTION
**Pull request by Mohsen-Awad**

### CODE REVIEW

File: task_1/index.html [hunk 1]
```diff
@@ -9,39 +9,137 @@
...
+            </div>
+            <div class="coffee-item">
+                <div class="image2"><img src="Double espresso.png" alt="MOCHA LATE"></div>
+                <div class="image2-text">
+                    <b><p>Double espresso</p></b>
+                    <p class="p-gray">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                </div>
+            </div>
         </div>
     </div>
```
Comment (ashrafabuodeh) [inline task_1/index.html:141]: remove space here

File: task_1/index.js [hunk 2]
```diff
@@ -1 +1,23 @@
...
+    console.log(x.id);
+    console.log(x.className);
+
+    if (x.className == "header-item") {
+
+        var z = document.getElementsByClassName("header-item");
+        for (var i = 0; i < z.length; i++)
+            z[i].classList.remove("active");
+
+
```
Comment (ashrafabuodeh) [inline task_1/index.js:16]: remove space

File: task_1/style.css [hunk 3]
```diff
@@ -4,27 +4,66 @@
...
+    width: 90px;
+}
+
 .main {
-    border: 1px solid;
     display: flex;
     margin-top: 20px;
-    padding-bottom: 32px;
+    padding-bottom: 10px;
 }
```
Comment (ashrafabuodeh) [inline task_1/style.css:59]: remove space

**Review by ashrafabuodeh — CHANGES_REQUESTED** [review]

File: task_1/index.html [hunk 4]
```diff
@@ -9,39 +9,137 @@
...
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
-            <div class="coffee-item"></div>
+            <div class="coffee-item">
+                <div class="image2"><img src="MOcha Late 02.png" alt="MOCHA LATE"></div>
```
Comment (araraltawarah98) [inline task_1/index.html:84]: let img tag down div tag.

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/style.css [hunk 5]
```diff
@@ -43,17 +82,78 @@
...
     width: 217px;
 }
 
 .grid-items {
-    border: 1px solid;
     display: grid;
-    grid-gap: 32px;
+    grid-gap: 64px 32px;
     grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
-    margin-top: 20px;
```
Comment (abood454) [inline task_1/style.css:57]: use shorthand margin

File: task_1/index.js [hunk 6]
```diff
@@ -1 +1,23 @@
-console.log("Hello World");
+
+const onMenuItemClick = (elementid) => {
+    console.log("Hello World");
+    var x = document.getElementById(elementid);
+
+    console.log(x);
+    console.log(x.id);
+    console.log(x.className);
+
```
Comment (abood454) [inline task_1/index.js:9]: remove space

**Review by abood454 — CHANGES_REQUESTED** [review]

**Review by ashrafabuodeh — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

**Review by abood454 — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/index.html [hunk 7]
```diff
@@ -1,47 +1,140 @@
...
     <link rel="stylesheet" href="style.css">
 </head>
-
-<body>   
+<body> 
     <div class="wrapper">
-        <div class="header"></div>
+        <div class="header">
+            <div class="coffee-logo">
+                <div><img class="coffee-image-logo" src="./photo/coffee logo.png" alt="coffee logo"></div>
```
Comment (mmasaid) [inline task_1/index.html:14]: use kabab case for image name, ids and class
```suggestion
                <div><img class="coffee-image-logo" src="./photo/coffee-logo.png" alt="coffee logo"></div>
```

File: task_1/index.html [hunk 8]
```diff
@@ -1,47 +1,140 @@
...
+            <div class="navbar">
+                <div id="home-item" class="header-item active" onclick="onMenuItemClick('home-item')">Home</div>
+                <div id="about-item" class="header-item" onclick="onMenuItemClick('about-item')">About Us</div>
+                <div id="lands-item" class="header-item" onclick="onMenuItemClick('lands-item')">Our Lands</div>
+                <div id="suppliers-item" class="header-item" onclick="onMenuItemClick('suppliers-item')">Our Suppliers</div>
+                <div id="products-item" class="header-item" onclick="onMenuItemClick('products-item')">Products</div>
+                <div id="contact-item" class="header-item" onclick="onMenuItemClick('contact-item')">Contact Us</div>
+            </div>
+            <div class="header-images">
+                <div><img class="shopping-image" src="./photo/shopping cart.png" alt="shopping cart"></div>
```
Comment (mmasaid) [inline task_1/index.html:25]: use kabab case for image name, ids and class

apply it everywhere 
```suggestion
                <div><img class="shopping-image" src="./photo/shopping-cart.png" alt="shopping cart"></div>
```

File: task_1/index.html [hunk 9]
```diff
@@ -1,47 +1,140 @@
...
+                    </div>
+                </div>
+            </div>
             <div class="hero-banner">
                 <div class="text-banner">
-                    <p>Be Active</p>
-                    <p>Barnsley Brew Coffee</p>
-                    <p>Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species. When coffee berries turn from.</p>
-                    <p>Top Packs</p>
+                    <p class="first-title">Be Active</p>
```
Comment (mmasaid) [inline task_1/index.html:43]:
```suggestion
                    <h4 class="item-main-title">Be Active</h4>
```

File: task_1/index.html [hunk 10]
```diff
@@ -1,47 +1,140 @@
...
+                    <h1 class="second-title">Barnsley Brew Coffee</h1>
+                    <p class="second-title-p">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species. When coffee berries turn from.</p>
+                    <p class="third-title">Top Packs</p>
```
Comment (mmasaid) [inline task_1/index.html:46]: I think if you use sub-title will be better 
```suggestion
                    <h1 class="second-title">Barnsley Brew Coffee</h1>
                    <p class="second-title-p">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species. When coffee berries turn from.</p>
                    <p class="third-title">Top Packs</p>
```

File: task_1/index.js [hunk 11]
```diff
@@ -1 +1,8 @@
-console.log("Hello World");
+const onMenuItemClick = (elementid) => {
+    let chosen_item = document.getElementById(elementid);
+    let header_items = document.getElementsByClassName("header-item");
+    for (let i = 0; i < header_items.length; i++)
+        header_items[i].classList.remove("active");
+
+    chosen_item.classList.add("active");
+}
```
Comment (mmasaid) [inline task_1/index.js:8]: 👍

File: task_1/style.css [hunk 12]
```diff
@@ -43,17 +80,76 @@
...
+    margin: 80px 0 0 0;
     padding: 3rem 0;
 }
+
+.black-coffee-text {
+    text-align: center;
+    width: 320px;
+}
+
+.distance {
```
Comment (mmasaid) [inline task_1/style.css:101]:
```suggestion
.price-container {
```

**Review by mmasaid — CHANGES_REQUESTED** [review]
Please address the comments and let me know if you have any question

**Review by mmasaid — APPROVED** [review]
LGTM

**Pull request merged**